### ISSUE TITLE
No metrics in lens 

### ISSUE THREAD

**Issue by jaccosteur** [lensapp/lens#3345]
**Describe the bug**
I do not see any metrics appear. I have windows 10 and a k3s deployment on 2 raspberry pi 4 8GB. I do see all the other information like pods, services etc.

**To Reproduce**
1. Install lens as described
2. In lens goto Catalog->Cluster
3. Hit the + key and add the k3s config file.
3. Connect to the k3s cluster.
4. Right click the k3s cluster and select settings
5. Under Metrics select Lens
6. Under Lens Metrics select the three options and apply.

**Expected behavior**
I would expect to see Cluster metrics in lens but there is nothing. 

**Screenshots**

**Environment (please complete the following information):**
- Lens Version: 5.0.2-latest.20210705.2
- OS: Windows
- Installation method (e.g. snap or AppImage in Linux): download installer

**Logs:**
When you run the application executable from command line you will see some logging output. Please paste them here:
```
In the logs I see:
[Metrics]: metrics not available {"error":{"name":"RequestError","message":"Error: socket hang up","cause":{"code":"ECONNRESET"},"error":{"code":"ECONNRESET"},"options":{"timeout":0,"resolveWithFullResponse":false,"json":true,"method":"POST","form":{"query":"sum(kube_node_status_allocatable{node=~\"m1\", resource=\"memory\"}) by (component)","start":"1625902140","end":"1625905740","step":"60","kubernetes_namespace":""},"headers":{"Host":"26bbb8e3e69bb0ba08ed56a960f89098.localhost:55052"},"uri":"http://localhost:55052/api-kube/api/v1/namespaces/lens-metrics/services/prometheus:80/proxy/api/v1/query_range","simple":true,"transform2xxOnly":false}}}
```

**Kubeconfig:**
Quite often the problems are caused by malformed kubeconfig which the application tries to load. Please share your kubeconfig, remember to remove any secret and sensitive information.
```
apiVersion: v1
clusters:
- cluster:
    certificate-authority-data: <DATA> 
    server: https://192.168.178.51:6443
  name: default
contexts:
- context:
    cluster: default
    user: default
  name: default
current-context: default
kind: Config
preferences: {}
users:
- name: default
  user:
    client-certificate-data: <DATA>
    client-key-data: <DATA>
```

**Additional context**

**Comment by jim-docker** [comment]
Hi, can you confirm that Prometheus is actually running?

**Comment by jaccosteur** [comment]
Hi, 
Yes prometheus is running. As mentioned I am running k3s with two nodes. I decided to drain the second (not the control plane node ofcourse). When running 1 node for some reason the metrics are now visible as expected. Looks cool by the way. I wonder why it is not working with two nodes. I used the prometheus and node exporter option within Lens under settings to deploy prometheus, kube state metric and node exporter on the k3s cluster.

After adding the second node, it keeps working. 

Not sure if adding two nodes right away caused the issue... I will rebuild and let you you know if it happens again. 

Thanks for responding.
